backend/python/app/graphql/view.py

```diff
@@ -14,7 +14,7 @@ def dispatch_request(self):
         data = self.parse_body()
 
         # The following mutations MUST request refreshToken
-        refresh_token_mutations = ["Refresh", "Login", "SignUp"]
+        refresh_token_mutations = ["Refresh", "Login"]
```

Comment:
hmmmm shouldnt we still require SignUp to request for refreshtoken? otherwise the response cookie wont get set